jigasi/Dockerfile

```diff
@@ -14,6 +14,23 @@ RUN apt-dpkg-wrap apt-get update && \
     apt-dpkg-wrap apt-get install -y jigasi jq && \
     apt-cleanup
 
-COPY rootfs/ /
+COPY --chown=noroot:noroot rootfs/ /
+
+RUN mkdir /config
+RUN mkdir -pm777 /tmp/transcripts
```

Comment:
Maybe we should change this to /transcripts or better yet /config/transcripts while we're at it.

Reply:
done. all paths have been updated to /config/transcripts